src/server/services/tests/registrar.getWeekNumber.test.js

```diff
@@ -0,0 +1,91 @@
+import registrar from '../registrar';
+
+it('Returns correct week number', async () => {
+  registrar.call = jest.fn().mockResolvedValue({
+    termSessionsByWeek: [
+      {
+        sessionTermCode: '20S',
+        termSessionCollection: [
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '0',
+            sessionWeekStartDate: '2020-03-21',
+            sessionWeekLastDate: '2020-03-27',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '1',
+            sessionWeekStartDate: '2020-03-28',
+            sessionWeekLastDate: '2020-04-03',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '2',
+            sessionWeekStartDate: '2020-04-04',
+            sessionWeekLastDate: '2020-04-10',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '3',
+            sessionWeekStartDate: '2020-04-11',
+            sessionWeekLastDate: '2020-04-17',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '4',
+            sessionWeekStartDate: '2020-04-18',
+            sessionWeekLastDate: '2020-04-24',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '5',
+            sessionWeekStartDate: '2020-04-25',
+            sessionWeekLastDate: '2020-05-01',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '6',
+            sessionWeekStartDate: '2020-05-02',
+            sessionWeekLastDate: '2020-05-08',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '7',
+            sessionWeekStartDate: '2020-05-09',
+            sessionWeekLastDate: '2020-05-15',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '8',
+            sessionWeekStartDate: '2020-05-16',
+            sessionWeekLastDate: '2020-05-22',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '9',
+            sessionWeekStartDate: '2020-05-23',
+            sessionWeekLastDate: '2020-05-29',
+          },
+          {
+            sessionCode: 'RG',
+            sessionWeekNumber: '10',
+            sessionWeekStartDate: '2020-05-30',
+            sessionWeekLastDate: '2020-06-05',
+          },
+        ],
+      },
+    ],
+  });
+
+  const responseForApril7 = await registrar.getWeekNumber('20S', '04/07/2020');
+  expect(responseForApril7).toEqual('2');
+
+  const responseForMay22 = await registrar.getWeekNumber('20S', '05/22/2020');
+  expect(responseForMay22).toEqual('8');
+
+  const responseForMay23 = await registrar.getWeekNumber('20S', '05/23/2020');
+  expect(responseForMay23).toEqual('9');
+
+  const responseForJune5 = await registrar.getWeekNumber('20S', '06/05/2020');
+  expect(responseForJune5).toEqual('10');
```

Comment:
@chern You mentioned the case in which the date is beyond Week 10. How does this code handle it? It should see if the date falls within 1 week after the end of 10th week and return Finals Week. Else return null.